app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 0.666)
```

Comment:
I think it's better to use expression `2 / 3` as a constant. It's easier to perceive and also a bit more accurate.